tests/python/test_dicts_lists.py

```diff
@@ -126,3 +126,18 @@ def test_eval_objects_jsproxy_delete():
 def test_eval_objects_jsproxy_compare():
     proxy = pm.eval("({a: 1, b:2})")
     assert proxy == {'a': 1.0, 'b': 2.0}
+
+def test_eval_objects_jsproxy_contains():
+    pm.eval("let obj = {'c':5}")
+    a = pm.eval('obj')
```

Comment:
```suggestion
    a = pm.eval('{'c':5}')
```

Reply:
This change would ensure that the object remains rooted as long as `a` exists, since as written the object will be permanently rooted since its declared and assigned at global scope.